openmdao/solvers/scipy_gmres.py

```diff
@@ -91,15 +91,12 @@ def solve(self, rhs_mat, system, mode):
             self.system = None
 
             if info > 0:
-                msg = "ERROR in solve in '{}': gmres failed to converge " \
-                      "after {} iterations"
+                msg = 'FAILED to converge after max iterations'
                 print(msg.format(system.name, options['maxiter']))
-                msg = 'FAILED to converge after hitting max iterations'
                 #logger.error(msg, system.name, info)
             elif info < 0:
-                msg = "ERROR in solve in '{}': gmres failed"
+                msg = 'ERROR returned from GMRES'
                 print(msg.format(system.name))
```

Comment:
Guess I didn't really want to change what we were printing to the screen, so I added that back. Check it now.